### ISSUE TITLE
Simple transactions: add padded & recursive fee calculation?

### ISSUE THREAD

**Issue by rphair** [cardano-foundation/developer-portal#1322]
There are repeated postings of Cardano users getting values calculated too low from `calculate-min-fee` when either:
- submitting a fee of `0` or zero values for the transaction input amounts
- not re-submitting with the calculated fee to ensure it's the same as the proposed fee (i.e. submitting recursively until then).

The advice to both pad the input fee & submit conservative or actual values, so that CBOR can be realistically assigned and a workable transaction fee determined from the byte cost, has been given here & in other places, though still not addressed @CarlosLopezDeLara in your otherwise excellent recently rewritten instructions (https://developers.cardano.org/docs/get-started/cardano-cli/get-started/simple-transactions#calculating-transaction-fees-and-balancing-a-transaction):
- https://github.com/IntersectMBO/cardano-cli/issues/827#issuecomment-2213892807
- https://forum.cardano.org/t/issues-with-outbound-transactions-after-upgrading-to-cardano-node-9-1-1-incorrect-fee-calculation/136133/9
- https://discord.com/channels/826816523368005654/1229425972105838642/1285969004657381499

Given that this question has to be answered manually every time, I propose that we agree upon & include language (to @CarlosLopezDeLara's high standards) that includes this "recursive" fee calculation as constantly recommended to the community.

I believe that if users continue to encounter trouble they may be tempted to use `transaction build` instead of the advance fee calculation even when their security configurations don't support doing this safely (as per https://developers.cardano.org/docs/get-started/secure-workflow#why-cant-i-use-cardano-cli-transaction-build).

**Comment by CarlosLopezDeLara** [comment]
Yes, this is a good idea. The key thing is that using --fee 0 is a bad idea. It may produce a slightly smaller tx than the real one once we add the fees. We have two options 

Start with 0, get a fee, rebuild the tx, get fee again, sign, submit. 

Start with something like --fee 0.170000

**Comment by CarlosLopezDeLara** [comment]
Yes, this is a good idea. The key thing is that using --fee 0 is a bad idea. It may produce a slightly smaller tx than the real one once we add the fees. We have two options 

Start with 0, get a fee, rebuild the tx, get fee again, sign, submit. 

Start with something like --fee 0.170000

**Comment by rphair** [comment]
> Start with something like --fee 0.170000

The older version of the doc pages had fee `300000` lovelace which was well backed up by recommendations circulated in the community.

Given the process requires convergence from either direction, in case the "recursion" doesn't happen I think it's better for a manual CLI user to pay for an extra CBOR byte or two than for a Tx to fail for a user who sometimes can't diagnose the problem.
